addons/udes_security/data/res_users.xml

```diff
@@ -13,6 +13,7 @@
                 (4, ref('udes_security.group_can_add_trusted_user')),
                 (4, ref('udes_security.group_trusted_user')),
                 (4, ref('udes_security.group_edit_allowed_file_types')),
+                (4, ref('udes_security.group_edit_allowlist_redirect_domains')),
```

Comment:
Can we add this to `user_root` as well please? Technically doesn't need them, but good to keep it consistent. :)